Update es.po

DarkPsydeLord opened the pull request:
deje en esa linea para ponernos de acuerdo como traducir plugin...
 y en el segundo pull dejo mas avances

pd. si vez que se duplican cosas es por que esta red esta un poco loquilla llegando a casa seguire trabajando por que es un martirio usar esta conexion del trabajo gracias.

saludos.

DarkPsydeLord committed:
Update es.po

DarkPsydeLord committed:
Update es.po

DarkPsydeLord committed:
Update es.po

DarkPsydeLord committed:
Update es.po

Noxbru commented on locales/es.po:
```diff
@@ -114,108 +114,110 @@ msgstr "booleano"
 #: src/commands.py:336
 msgid "do that"
-msgstr ""
+msgstr "Haslo"
```
Corrección ortográfica: "Hazlo"

Noxbru commented on locales/es.po:
```diff
@@ -114,108 +114,110 @@ msgstr "booleano"
 #: src/commands.py:414 src/commands.py:417
 msgid "regular expression"
-msgstr "FALTA TRADUCCION"
+msgstr "expresion regular"
```
Corrección ortográfica: "expresión regular"

Noxbru commented on locales/es.po:
```diff
@@ -114,108 +114,110 @@ msgstr "booleano"
 #: src/commands.py:539
 msgid "You must not give the empty string as an argument."
-msgstr "FALTA TRADUCCION"
+msgstr "No debes usar un string vacio como argumento."
```
Corrección ortográfica: 'vacío'

Noxbru commented on locales/es.po:
```diff
@@ -114,108 +114,110 @@ msgstr "booleano"
-
+msgstr""
+"Determina si este complemente esta cargado\n"
+"         predeterminado"
```
Corrección: "Determina si este complemento está cargado por defecto"

SantiagoTorres replied:
Necesitamos un espacio entre `msgstr` y `""`

Noxbru commented on locales/es.po:
```diff
@@ -114,108 +114,110 @@ msgstr "booleano"
-
+msgstr ""
+"Determina si este complemento esta\n"
+"        visible publicamente."
```
Corrección ortográfica: "Determina si este complemento está visible públicamente"

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
+"   seran usados si el nick principal (supybot,nick) no esta disponible.\n"
+"   %s en este nick es reemplazado por el valor de supybot.nick en uso. si no\n"
+"   se encuentran alternativas, o si, todas estan en uso, supybot.nick sera usado\n"
+"   de la manera mas apropiada hasta encontrar un nick que no se encuentre en uso."
```
Correcciones ortográficas y cambios menores:
"Determina qué nicks alternativos\n"
"   serán usados si el nick principal (supybot.nick) no está disponible.\n"
"   En este nick, %s será sustituido por el valor de supybot.nick. Si no\n"
"   se encuentran alternativas, o si todas están en uso, supybot.nick será modificado\n"
"   apropiadamente hasta encontrar un nick que no se encuentre en uso."

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
-msgstr "FALTA TRADUCCION"
-
+msgstr""
+"Determina la intentacion de los strings del bot, si el servidor\n"
```
Corrección: 'intentacion' -> 'indentación'
Nota: No estoy seguro de la traducción, le he preguntado a Fundeu a ver si sugieren algo mejor. De todos modos, como poco falta una tilde.

Noxbru replied:
Corrección a la corrección: 'ident' es una palabra y significa 'identificativo'. Si no hay una errata en el original (ident por indent) la traducción sería más o menos:
"Determina el string de identificación del bot, si el servidor\n"
"   no provee de uno por defecto."

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
+msgstr ""
+"Determina el usuario que el bot envia al servidor.\n"
+"    Un usuario predeterminado usando la version actual del bot sera generado si\n"
+"    se esto se deja vacio."
```
Correcciones ortográficas/estilo:
"Determina el usuario que el bot envía al servidor.\n"
"   Se generará un usuario predeterminado utilizando la versión actual del bot si\n"
"   se deja en blanco."

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
 #: src/conf.py:231
 msgid "Determines what networks the bot will connect to."
-msgstr "FALTA TRADUCCION"
+msgstr "Determina a que red se conectara el bot."
```
Corrección ortográfica y gramatical:
"Determina a qué redes se conectará el bot."

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
+msgstr ""
+"Determina que contraseña sera usada en %s.   Si, sabemos que\n"
+"        tecnicamente los passwords son usados en el servidor especificamente y no\n"
+"        en la red, pedo esta es la mejor forma que podemos usar por el momento."
```
Correcciones varias:
"Determina qué contraseña será usada en %s.   Sí, sabemos que\n"
"   técnicamente las contraseñas son específicas del servidor y no de la red,\n"
"   pero esto es lo mejor que podemos hacer por el momento.

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
+msgstr ""
+"Determina a que servidores se conectara el bot para %s. cada uno\n"
+"        sera intentado en orden, regresando al principio cuando el ciclo es\n"
+"        completado."
```
Corrección ortográfica:
"Determina a qué servidores se conectará el bot para %s. Cada uno\n"
"   será intentado en orden, regresando al principio cuando se complete\n"
"   el ciclo."

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
 #: src/conf.py:281
 msgid "Determines what channels the bot will join only on %s."
-msgstr "FALTA TRADUCCION"
+msgstr "Determina a que canales se unira el bot en %s solamente."
```
Corrección ortográfica:
"Determina a qué canales se unirá el bot en %s solamente."

Noxbru commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
+msgstr ""
+"Determina\n"
+"        que nick debe usar el bot en esta red. si esto esta vacio de forma\n"
+"        predeterminada se usara el parametro ensupybot.nick"
```
Correcciones menores:
"Determina\n"
"   qué nick usará el bot en esta red. Si se deja en blanco, utilizará por defecto\n"
"   supybot.nick."

Noxbru commented on locales/es.po:
```diff
@@ -48,73 +48,73 @@ msgstr "Intentaste anidar más expresiones de las que estan permitidas actualmen
 #: src/callbacks.py:773
 msgid "The command %q is available in the %L plugins.  Please specify the plugin whose command you wish to call by using its name as a command before %q."
-msgstr "El comando %q está disponible en %L. Por favor especifica"
+msgstr "El comando %q está disponible en los complementos de %L. Por favor especifica el comando que quieres usar de este complemento usando su nombre como comando antes de %q"
```
Corrección de estilo:
"El comando %q está disponible en los complementos %L. Por favor, especifica el complemento cuyo comando deseas utilizar utilizando su nombre como si fuese un comando antes de %q."

Noxbru commented on locales/es.po:
```diff
@@ -48,73 +48,73 @@ msgstr "Intentaste anidar más expresiones de las que estan permitidas actualmen
 #: src/callbacks.py:916
 msgid "more messages"
-msgstr "FALTA TRADUCCION"
+msgstr "mensajes más"
```
En estos tres últimos quizás falta un poco de contexto para saber si esa es la traducción correcta o habría que invertir el orden de las palabras.

SantiagoTorres replied:
Creo que cuando veamos el comportamiento podemos marcarlo como bug. Por ahorita lo dejaria asi

DarkPsydeLord replied:
les confirmo la traduccion es literal ya que la linea de codigo dice:

`if not allowedLength: # 0 indicates this.
                        allowedLength = 470 - len(self.irc.prefix)
                        allowedLength -= len(msg.nick)
                        # The '(XX more messages)' may have not the same
                        # length in the current locale
                        allowedLength -= len(_('(XX more messages)'))
`
literalmente hablamos de una cantidad de mensajes más.

Noxbru commented on locales/es.po:
```diff
@@ -48,73 +48,73 @@ msgstr "Intentaste anidar más expresiones de las que estan permitidas actualmen
-"    nell'elenco dei comandi, ecc.; sarà come se non esistessero."
+msgstr ""
+"Determina que comandos están deshabilitados actualmente.\n"
+"    Estos comandos no aparecerán en la lista de comandos, dán la impresión de no existir."
```
Corrección menor: 'dan' sin tilde.

Noxbru commented on locales/es.po:
```diff
@@ -48,73 +48,73 @@ msgstr "Intentaste anidar más expresiones de las que estan permitidas actualmen
 #: src/commands.py:263
 msgid "non-negative integer"
-msgstr "FALTA TRADUCCION"
+msgstr "número entero no negativo."
```
Tanto aquí como en L84 como en L88 estás añadiendo un punto al final, tal vez no sea necesario.

SantiagoTorres commented on locales/es.po:
```diff
@@ -12,210 +12,212 @@ msgstr "FALTA TRADUCCION"
 #: src/callbacks.py:1231
 msgid "Invalid arguments for %s."
-msgstr "FALTA TRADUCCION"
+msgstr "Argumentos invalidos para %s."
```
inválidos*

SantiagoTorres commented on locales/es.po:
```diff
@@ -12,210 +12,212 @@ msgstr "FALTA TRADUCCION"
 #: src/commands.py:239
 msgid "non-integer value"
-msgstr "FALTA TRADUCCION"
+msgstr "el valor de un número que no sea entero."
```
Creo que esta frase se utiliza en otros contextos. Creo que vale la pena decir "número no-entero"

SantiagoTorres commented on locales/es.po:
```diff
@@ -12,210 +12,212 @@ msgstr "FALTA TRADUCCION"
 #: src/commands.py:336
 msgid "do that"
-msgstr "FALTA TRADUCCION"
+msgstr "Haslo"
```
Házlo en imperativo va con tilde y z

SantiagoTorres commented on locales/es.po:
```diff
@@ -224,64 +226,80 @@ msgid ""
 "    doesn't provide one by default."
-msgstr "FALTA TRADUCCION"
-
+msgstr""
```
aquí también falta espacio

SantiagoTorres reviewed (approved):
En general creo que las traducciones están bien :) Fuera de los comentarios de @Noxbru , y el problema de los espacios creo que vamos en buen camino 👍

DarkPsydeLord committed:
Update es.po

dejo pocos cambios y pido sus sugerencias para las siguientes palabras:
ya que yo normalmente no las traduzco a ninguna palabra gracias

palabra               sugerencia

Nick
Password
Plugin
bot

Noxbru commented on locales/es.po:
```diff
@@ -306,36 +306,43 @@ msgid ""
 #: src/conf.py:298
 msgid "Determines what SASL password will be used on %s."
-msgstr "FALTA TRADUCCION"
+msgstr "Determina que contraseña sera usada en %s."
```
Corrección ortográfica:
"Determina qué contraseña será usada en %s."

Noxbru commented on locales/es.po:
```diff
@@ -306,36 +306,43 @@ msgid ""
+msgstr ""
+"Determina que nombre de usuario SASL sera usado en %s. Debería ser\n"
+"       el nombre de la cuenta del bot. Debido a la forma en que SASL trabaja, no puede ser\n"
+"       usado ningun nick agrupado."
```
Correcciones ortográficas menores:
"Determina qué nombre de usuario SASL será usado en %s. Debería ser\n"
"   el nombre de la cuenta del bot. Debido a la forma en que SASL trabaja, no puede ser\n"
"   usado ningún nick agrupado."

Noxbru commented on locales/es.po:
```diff
@@ -306,36 +306,43 @@ msgid ""
+msgstr ""
+"Determina el tamaño maximo absoluto de\n"
+"    las respuestas del bot -- no pasaran las respuestas del bot con una longitud mayor\n"
+"    que esta."    
```
Correcciones ortográficas menores:
"Determina el tamaño máximo absoluto de\n"
"   las respuestas del bot -- no pasarán las respuestas del bot con una longitud mayor\n"
"   que esta."

Ligero detalle, dice 'pass through' que es atravesar, no que el bot esté respondiendo...

DarkPsydeLord replied:
el contexto es con como responde el bot en tamaño máximo de respuesta creo que esta dentro de la posible traducción pero con gusto reviso el codigo para tener la certeza de lo que se trata.

gracias por la anotación

DarkPsydeLord committed:
Update es.po

DarkPsydeLord commented on locales/es.po:
```diff
@@ -224,166 +226,207 @@ msgid ""
-""
+msgstr ""
+"Determina cuando el bot rompera lineas de mensaje\n"
+"    largas en diferentes partes para permitir a los usuarios usar el comando 'mas' para obtener"
```
por favor no acentuar los 'mas' ya que serán usados como comando por el bot en un futuro y creo que es preferible una palabra plana que una acentuada por supuesto esto lo pongo a su consideración. 

gracias.

Noxbru replied:
Dejaremos 'mas' como comando para el bot o conservaremos el original 'more' ?
Y en detalles ortográficos: 'rompera' -> 'romperá' y 'linea' -> 'línea'

DarkPsydeLord commented on locales/es.po:
```diff
@@ -224,166 +226,207 @@ msgid ""
+msgstr ""
+"Determina que tan larga será la parte de un mensaje\n"
+"    si es establecido a 0, usa nuestro super-retoque,\n"
+"    obtiene-el-maximo-de-un-mensaje-individual como predeterminado."
```
no estoy seguro si traduciré o no esta parte, estoy en revisión del codigo.

Noxbru replied:
Tiene pinta de ser algo para el dev, no para los usuarios.

Noxbru commented on locales/es.po:
```diff
@@ -349,66 +349,84 @@ msgid ""
 "    chunks (for use with the 'more' command) will be."
-msgstr "FALTA TRADUCCION"
+msgstr ""
+"Determina cual es el numero maximo de\n"
```
Corrección ortográfica: 'maximo' -> 'máximo'

Noxbru commented on locales/es.po:
```diff
@@ -349,66 +349,84 @@ msgid ""
+"Determina si el bot debe enviar\n"
+"    respuestas multi-mensaje en un solo mensaje o en multiples mensajes.\n"
+"    por motivos de seguridad (sera menos probable que el bot genere flood)\n"
+"    enviara normalmente todo en un solo mensaje, usando 'mas' si es necesario."
```
Detalles ortográficos: 'por' -> 'Por', 'sera' -> 'será', 'enviara' -> 'enviará'.

Noxbru commented on locales/es.po:
```diff
@@ -349,66 +349,84 @@ msgid ""
-""
+msgstr ""
+"Determina si el bot debe responde con un mensaje de error\n"
+"    cuando es mencionado sin un comando valido. Si este valor es Falso, el bot permanecerá\n"
```
Detalle ortográfico: 'valido' -> 'válido'

Noxbru commented on locales/es.po:
```diff
@@ -349,66 +349,84 @@ msgid ""
+msgstr ""
+"Determina si los mensajes de error que son resultado\n"
+"    de fallos en el bot deben ser mostrados como mensajes de error detallados\n"
+"    (las excepciones sin capturar) o un mensaje de error generico."
```
Detalle ortográfico: 'generico' -> 'genérico'

Noxbru commented on locales/es.po:
```diff
@@ -349,66 +349,84 @@ msgid ""
+msgstr ""
+"Determina si el bot debe enviar un mensaje de error\n"
+"    a los usuarios en privado. Deberias hacer esto para mantener el trafico del canal\n"
+"    en el minimo. Puede ser usado en combinacion con supybot.reply.error.withNotice."
```
Detalles ortográficos: 'Deberias' -> 'Deberías', 'trafico' -> 'tráfico', 'minimo' -> 'mínimo', 'combinacion' -> 'combinación'
